# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mapped_hyph/src/bin/hyf_compile.rs
# Commit: bc5b6dcb7057
# Full Hash: bc5b6dcb7057cf610efe29e14b65e9c3aa34d1c3
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-10-12 15:27:19
# Description:
#   Bug 1669162 - Update mapped_hyph to 0.4.2 so that .dic parse errors are non-fatal. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D92435
# ==============================================================================

diff -r a083e04138aa -r bc5b6dcb7057 third_party/rust/mapped_hyph/src/bin/hyf_compile.rs
--- a/third_party/rust/mapped_hyph/src/bin/hyf_compile.rs	Mon Oct 12 09:52:22 2020 +0000
+++ b/third_party/rust/mapped_hyph/src/bin/hyf_compile.rs	Mon Oct 12 10:23:27 2020 +0000
@@ -8,11 +8,13 @@
 // except according to those terms.
 
 extern crate mapped_hyph;
+extern crate env_logger;
 
 use std::env;
 use std::fs::File;
 
 fn main() -> std::io::Result<()> {
+    env_logger::init();
     let args: Vec<String> = env::args().collect();
     if args.len() == 3 {
         let in_file = File::open(&args[1])?;